src/errors.d

```diff
@@ -18,6 +18,8 @@ import ddmd.globals;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 
+//debug = printGaggedErrors;
```

Comment:
No. There are thousands of debugging printf's in the source code, and they are individually turned on/off by deleting/adding // prefixes. Assigning names to them makes sense only if there are groups of related ones spread throughout the file. This is not the case here. Furthermore, blocks can be turned on/off with `version(all)` / `version(none)`. Again, the extra name here 200 lines away is not justified.